internal/generator/fluentd/output/loki/bearer-token-file.go

```diff
@@ -0,0 +1,18 @@
+package loki
+
+import (
+	"github.com/openshift/cluster-logging-operator/internal/generator/fluentd/output/security"
+)
+
+type BearerTokenFile security.BearerTokenFile
```

Comment:
Same here: global search/replace BearerTokenFile with BearerToken.

Reply:
@alanconway we just point to the file path. It doesn't contain the actual token.
The secret will look like:
`token=/var/run/secrets/kubernetes.io/serviceaccount/token`